src/RoomsSDKAdapter.js

```diff
@@ -126,12 +135,56 @@ export default class RoomsSDKAdapter extends RoomsAdapter {
       );
 
       // Convert to a multicast observable
-      this.getRoomObservables[ID] = getRoom$.pipe(
-        publishReplay(1),
-        refCount(),
-      );
+      this.getRoomObservables[ID] = getRoom$.pipe(publishReplay(1), refCount());
     }
 
     return this.getRoomObservables[ID];
   }
+
+  /**
+   * Returns an observable that emits current and future activities from the specified room.
+   *
+   * @param {string} id ID of room to get
+   * @returns {Observable.<Activity>} Observable stream that emits current and future activities from the specified room
+   */
+  getRoomActivities(id) {
+    logger.debug('ROOM', id, 'getRoomActivities()', ['called with', {id}]);
+
+    // if (!(ID in this.getRoomObservables)) {
+    //   return throwError(new Error(`Room ${ID} does not exist`));
+    // }
+
+    const activity$ = new BehaviorSubject(null);
+
+    this.datasource.internal.mercury.on(
+      'event:conversation.activity',
+      (activity) => {
+        const {id: hydraId} = deconstructHydraId(id);
```

Comment:
Well, this is not really a hydra ID anymore, this is the UUID at this point.
```suggestion
        const {id: UUID} = deconstructHydraId(id);
```